### PR TITLE
add cucumber to the left sidebar and cucumber config page

### PR DESCRIPTION
**Pull request by valentyn88**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/cucumber.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+id: cucumber
+title: "Cucumber"
+sidebar_label: Cucumber
+---
+
+import Highlight from '../../../src/components/highlight.jsx'
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cucumber.md:7]: This import is no longer needed, as we don't use that highlight tag anymore.

**Review by sweeneyskirt-sl — APPROVED** [review]
I guess that's what I get for missing standup. I am not sure I agree with this solution, since `cucumber` is not actually a framework, so it seems out of place in this space in the nav, but I understand that it has its own set of configs that must be documented so for now, it is fine.

File: docs/testrunner-toolkit/configuration/cucumber.md (hunk 1 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/cucumber.md:7]: good to know, I can remove it.

**Comment by valentyn88** [comment]
> I guess that's what I get for missing standup. I am not sure I agree with this solution, since `cucumber` is not actually a framework, so it seems out of place in this space in the nav, but I understand that it has its own set of configs that must be documented so for now, it is fine.

Maybe it's not the best place. We can reorganize it, but it should be documented.

**Pull request merged**